sidebar/lib/mute.js

```diff
@@ -0,0 +1,31 @@
+var muteEnabled = false;
+
+function wrapper(id, ci, tab) {
+  muteTab(tab);
+}
+
+function muteTab(tab) {
+  browser.tabs.update(tab.id, { muted: true });
+}
+
+function toggleMute() {
```

Comment:
Can we move this functionality into a panelshutup.js instead of creating a new lib directory?

Reply:
Also change names to shutup vs. mute.